### ISSUE TITLE
High severity vulnerabilities - Newtonsoft, System.Security.Cryptography.Pkcs

### ISSUE THREAD

**Issue by martinbryant** [Zaid-Ajaj/Fable.Remoting#363]
Package 'Newtonsoft.Json' 12.0.2 has a known high severity vulnerability, https://github.com/advisories/GHSA-5crp-9r3c-p9vr

Package 'System.Security.Cryptography.Pkcs' 6.0.1 has a known high severity vulnerability, https://github.com/advisories/GHSA-555c-2p6r-68mm

It looks like Giraffe needs updating to 6.x to be able to get Newtonsoft 13.x

**Comment by RicoSaupe** [comment]
will there be an update to this? Mend scanner is also recognizing this. 

![image](https://github.com/Zaid-Ajaj/Fable.Remoting/assets/145867813/9335d5a8-fadc-416a-a394-20d7b42dd056)

**Comment by kerams** [comment]
I'm not sure how this is a Fable.Remoting concern. There is nothing preventing you from bumping Giraffe as far as I can see.

**Comment by RicoSaupe** [comment]
In my case its not about giraffe. its about fable remoting using the "older" Newtonsoft library and the request to bump this up to the latest version

**Comment by kerams** [comment]
Sorry, but the argument still stands

![image](https://github.com/Zaid-Ajaj/Fable.Remoting/assets/5063478/c6fef19c-fbe6-45c6-b920-6d376f86416e)

**Comment by RicoSaupe** [comment]
Yes. I understand that i can use a higher version. Just wondering about the reason of the 12.x version. Is it for compatibility?

**Comment by Zaid-Ajaj** [comment]
> Just wondering about the reason of the 12.x version. Is it for compatibility?

@RicoSaupe We can update it, I don't think there is a reason not to
